src/matrix/ssss/SecretStorage.ts

```diff
@@ -37,7 +43,7 @@ export class SecretStorage {
         }
     }
 
-    async _decryptAESSecret(type, encryptedData) {
+    async _decryptAESSecret(type: string, encryptedData: any): Promise<string> {
```

Comment:
perhaps we can type encrypted data (and cast `accountData?.content?.encrypted?.[this._key.id]` to it above), it's defined here https://github.com/uhoreg/matrix-doc/blob/symmetric_ssss/proposals/2472-symmetric-ssss.md and should look like this:
```json
{
  "type": "m.megolm_backup.v1",
  "content": {
    "encrypted": {
      "the key id": {
        "iv": "...",
        "ciphertext": "...",
        "mac": "..."
      }
    }
  }
}
```

Reply:
But EncryptedData only needs to be:
```json
 {
        "iv": "...",
        "ciphertext": "...",
        "mac": "..."
}
```
?